java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/jdbc/PreparedStatement.java

```diff
@@ -272,13 +272,12 @@ protected void dataBuild() throws SQLException {
                 columnTypes.AddColumnType(types.get(i + 1));
             }
             this.currentRow = SQLRequestRow.CreateSQLRequestRowFromColumnTypes(columnTypes);
-            // TODO(hw): check if null
+        if (this.currentRow == null) {
+        	throw new SQLException("failed to build data with row");
+        }
```

Comment:
hi @vighnesh-kadam  the line of 275 and 277  also need to add indentation

Reply:
i used the deafault format option in eclipse(as suggested by contribution guidelines) but it changed the formatting of the entire page so i have done it manually. Please correct me if i'm doing it wrong.

Reply:
good job!  And `throw` statement looks  indenting more space, it also need to remove redundant space before `throw` word. thank you

Reply:
my new indentation pr is correct right?   i had a doubt , when i used the eclipse default formatter (as given in contribution guidelines) if changed the entire page's formatting so instead i had to do it manually,  is there any other solution to that?

Reply:
ok i will do it

Reply:
![WX20220511-152322](https://user-images.githubusercontent.com/6183996/167792518-fd9e0203-ea37-4fbe-906d-29fbbedff45a.png)
it seems having more space compared with other statement.

Reply:
there is `Tab` before `throw` ? replace it with space

Reply:
yes its done now